src/workers/_inputs.js

```diff
@@ -376,3 +443,34 @@ export function fetchGenes() {
         return cache.genes;
     }
 }
+
+export function fetchAnnotations(col) {
+    let annots;
+    if ("reloaded" in cache) {
+        annots = cache.reloaded.annotations;
+    } else {
+        annots = cache.annotations;
+    }
+
+    if (!annots.headers.includes(col)) {
+        throw `column ${col} does not exist in col.tsv`;
+    }
+
+    let colIdx = annots.headers.indexOf(col);
+    let uvals = [];
+    let vals = annots.rows.map(x => {
```

Comment:
See comment above about paying the columnar organization cost up-front instead of at every fetch.

Reply:
Also suggest using a `Uint32Array` here, so that the buffer can be transferred.